spec/rubocop/cop/rails/reversible_migration_spec.rb

```diff
@@ -169,6 +169,10 @@ def change
       remove_foreign_key :accounts, :branches
     RUBY
 
+    it_behaves_like 'accepts', 'remove_foreign_key(with table option)', <<~RUBY
```

Comment:
Would you update with the description using `:to_table`? This suggestion is less than 80 characters :-)

```suggestion
    it_behaves_like 'accepts', 'remove_foreign_key(with `:to_table`)', <<~RUBY
```